capstone-app/src/app/modules/maps/map/map.component.ts

```diff
@@ -89,12 +88,45 @@ export class MapComponent implements OnInit{
     this.activeMarkerOutput.emit(this.activeMark);
   }
 
+  getCurrentLocation () {
+    let filterResult: string;
+    let prevResult: string = 'blank';
+    navigator.geolocation.getCurrentPosition(position => {
+      this.position = {
+        lat: position.coords.latitude,
+        lng: position.coords.longitude,
+      };
+      this.location = new google.maps.LatLng(this.position);
+      this.geocoder.geocode({'location': this.position}, (results, status) => {
+        if (status === 'OK') {
+          results.filter(result => {
+          const addressSizeCount = result.formatted_address.split(',').length;
+          if (addressSizeCount == 3 && !result.formatted_address.includes('County') && !result.formatted_address.match(/\d+/g)) {
```

Comment:
Will the address size count always be 3? Even in edge cases like Vatican City, Washington D.C., etc